### DISCUSSION TITLE
[FR]:Change column copy to Wikilink.

### DISCUSSION THREAD

**Discussion by kazdonkai** [RafaelGB/obsidian-db-folder#662] [Ideas]
### Present your request

Currently, right-clicking on a column and copying copies the external link, which is not user-friendly.
So, could you please consider changing copy to wikilink or give us the option to change it to wikilink.

### For which platform do you request this request??

Desktop

**Comment by kazdonkai** [comment]
Maybe the problem is related to [FR] Change drag & drop of internal links behavior #43.

**Reply by RafaelGB** [reply]
It's the native menu, I do not dev a particular one. Once the UI and usability of some menus is improved I will try to check if the API allows us to manage the menu

Moved to idea discussion
